app/dash/Notify/index.js

```diff
@@ -8,75 +8,11 @@ import { usesBaseFee } from '../../../resources/domain/transaction'
 
 import Confirm from '../../../resources/Components/Confirm'
 import AddToken from './AddToken'
-import frameIcon from '../../../asset/FrameIcon.png'
 
 const FEE_WARNING_THRESHOLD_USD = 50
 const capitalize = (s) => s[0].toUpperCase() + s.slice(1)
 
 class Notify extends React.Component {
-  betaDisclosure() {
```

Comment:
I think we still reference this code below

Reply:
good catch